test.js

```diff
@@ -447,3 +451,51 @@ test('alias - require previousId and userId', t => {
     })
   })
 })
+
+test('e2e', async t => {
+  const id = uid(16)
+
+  // Segment Write Key for https://segment.com/segment-libraries/sources/analytics_node_e2e_test/overview
+  const analytics = new Analytics('wZqHyttfRO0KxEHyRTujWZQswgTDZx1N')
+  analytics.track({
+    userId: 'prateek',
+    event: 'E2E Test',
+    properties: {
```

Comment:
=> `properties: { id } `

Reply:
That should be a thing that should be flagged by the style guide/standard. If it didn't, i don't think it's worth bothering with those changes.

Reply:
It's about consistency. Here it's on multiple lines and on the other ones it's on one (https://github.com/segmentio/analytics-node/pull/134/files#diff-1dd241c4cd3fd1dd89c570cee98b79ddR474, https://github.com/segmentio/analytics-node/pull/134/files#diff-1dd241c4cd3fd1dd89c570cee98b79ddR476).

Reply:
I think that's okay. The point of code formatter is to not have to make these kinds of changes by hand. If a style guide doesn't flag it, I think it's okay to leave it as creative expression :)

Reply:
(though I did update it here as I was making other changes at the same time).